public/app/features/templating/dataMacros.test.ts

```diff
@@ -101,7 +101,7 @@ describe('dataMacros', () => {
 
     const scopedVars = { __dataContext: dataContext };
 
-    expect(_templateSrv.replace('${__data.fields[1]}', scopedVars)).toBe('10000');
+    expect(_templateSrv.replace('${__data.fields[1]}', scopedVars)).toBe('1969-12-31 18:00:10.000');
```

Comment:
Is this related to the PR? Why does this value change?

Reply:
Field 1 in this test is a date, with the type of `FieldType.date` https://github.com/grafana/grafana/blob/main/public/app/features/templating/dataMacros.test.ts#L26

The values of this field are `5000` and `10000` which are interpreted as timestamps now that the field type is passed in. I'm guessing this timestamp is 10,000 seconds from epoch.